schemas/json/README.md

```diff
@@ -58,24 +58,35 @@ We explicitly forbid empty JSON arrays to avoid confusion about attributes which
 Namely, an empty array is semantically equal to an omitted attribute (according to the meta-model).
 Thus, the JSON property representing an aggregation attribute must be omitted if the aggregation is empty.
 
-For better readability, we diverge from the meta-model and spell most properties in plural.
+In UML, it is the convention to name associations and aggregations in singular form. The cardinality is to be taking into account to decide on whether there are none, a single or several elements in the corresponding association or aggregation. 
+In JSON it is the array that declares the set of elements, therefore we choose the plural name for it. The singular name is used for its descriminator (see section on decriminators).
```

Comment:
An array declares a sequence of elements? I already rewrote this section.

Reply:
Now it is: In UML, it is the convention to name associations and aggregations in singular form. 
The cardinality is to be taken into account to decide on whether there are none, a single or several elements in the 
corresponding association or aggregation. 
In JSON it is best practice to use plural form for array in class properties. 
The singular name is used for its descriminator (see section on decriminators).
Typically the plural name is derived by just adding an "s" to the name.